gammapy/makers/safe.py

```diff
@@ -170,6 +170,12 @@ def make_mask_energy_aeff_max(self, dataset, observation=None):
             position = geom.center_skydir
 
         aeff = exposure.get_spectrum(position) / exposure.meta["livetime"]
+        if not np.any(aeff.data) > 0.1:
```

Comment:
We should definitely avoid the "magic number" here, because `aeff` could be in any units. Isn't the non-zero requirement enough?

Reply:
A typo, I wanted to write 0!
Thank you for your accurate attention